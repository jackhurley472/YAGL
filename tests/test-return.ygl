printString(sreturned(55));
String sreturned(int x) {
	printInt(x);
	return "Hello World Returned!";
}

int ireturned(int x) {
	return x;
}

printInt(ireturned(5559));
