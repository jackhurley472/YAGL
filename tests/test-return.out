55
Hello World Returned!
5559
